src/organizations/components/OrgProfileTab/DeletePanel.tsx

```diff
@@ -1,44 +1,81 @@
 // Libraries
-import React, {FC, useContext} from 'react'
```

Comment:
This file was getting too large and confusing. Instead, I refactored it into three components:

- DeletePanel: either shows the Delete Account button in a free account with one org; or Delete Org in any account with multiple orgs.
- Delete Account button
- Delete Org button